src/UI/images/Image.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from '../../styles.scss';
+
+export default function Image({ image }) {
+  return (
+    <div className={styles.images}>
+      <img src={image}></img>
```

Comment:
you can use` <img />` if there are no children. Also, it's always good to add an alt attribute to images  https://www.w3schools.com/tags/att_img_alt.asp

Reply:
for some reason I cant close this conversation